controllers/users.js

```diff
@@ -402,6 +401,15 @@ var deleteUser = function(id, callback) {
   });
 };
 
+const validatePassword = password => {
+  if (password.length < PASSWORD_MINIMUM_LENGTH) {
+    return error400(`The password must be at least ${PASSWORD_MINIMUM_LENGTH} characters long.`);
```

Comment:
I don't think so and none of them are. These aren't shown in the webapp anywhere just in the command line.